### DISCUSSION TITLE
How to run oemer?

### DISCUSSION THREAD

**Discussion by Mimrix** [BreezeWhite/oemer#24] [Q&A] [answered]
Hello, I'm struggling with running oemer. How to run it? Where to put oemer <path_to_image>? In cmd, it doesn't recognize the command. And ete.py isn't opening. It blinks only for a sec. So how to use the program? The readme is very, very unclear about it... thanks for your response.

**Comment by BreezeWhite** [comment, marked as answer]
Seems like you are fresh to the python world. The section `Quick Start` basically describes how to install and use `oemer` command by command. Instead installing on your own machine, I would suggest you using [colab](https://colab.research.google.com/github/BreezeWhite/oemer/blob/main/colab.ipynb). You just need to press the 'execute' button one by one and follow the instructions, and you should be able to get what you want.

**Reply by Mimrix** [reply]
Thank you very much...
